lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -747,6 +747,22 @@ def _validate_favorite_object_type(self, object_type):
                 f"This type is not supported. Given object_type: {object_type}"
             )
 
+    @expose_api
+    def set_theme(self, trans, id: str, theme: str, payload=None, **kwd) -> str:
```

Comment:
We might want to consider a different approach to preference setting like this down the road.  

i.e. -- do we really need a route for every defined preference?  Or, can we just use a single flexible 'preferences' endpoint that takes a key/value pair?

Reply:
We should also stop adding non-fastAPI routes.